tests/unit/test_queries.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+import pytest
+
+QUERIES = [
+    f"q{i}a.sql" if i in {14, 23, 24, 39} else f"q{i}.sql" for i in range(1, 100)
+]
+
+
+@pytest.fixture(scope="module")
+def c():
+    # Lazy import, otherwise the pytest framework has problems
+    from dask_sql.context import Context
+
+    c = Context()
+    for table_name in os.listdir(f"{os.path.dirname(__file__)}/data/"):
+        c.create_table(
+            table_name,
+            f"{os.path.dirname(__file__)}/data/{table_name}",
+            format="parquet",
+            gpu=False,
+        )
+
+    yield c
+
+
+@pytest.fixture(scope="module")
+def gpu_c():
+    pytest.importorskip("dask_cudf")
+
+    # Lazy import, otherwise the pytest framework has problems
+    from dask_sql.context import Context
+
+    c = Context()
```

Comment:
Even though it's single gpu, I'm inclined to try using a local cluster/cudacluster to test this with the distributed scheduler. Though if that's tougher to incorporate, this is a good starting point.